[Term Entry] PHP Math Functions deg2rad() (#2469)
Codecademy/docs: content/php/concepts/math-functions/terms/deg2rad/deg2rad.md

---
Title: 'deg2rad()'

> Description: 'Converts the number in degrees to the radian equivalent'

Reviewer: ```suggestion
Description: 'Converts the number in degrees to the radian equivalent.'
```

Author: Done

> Description: 'Converts the number in degrees to the radian equivalent.'

Reviewer 2: ```suggestion
Description: 'Converts a number in degrees to the radian equivalent.'
```

Subjects:
  - 'Computer Science'
  - 'Web Development'

>   - 'Math Functions'

Reviewer 2: ```suggestion
```

Reviewer 2: This not currently a subject we use.

Tags:
  - 'Arithmetic'
  - 'Functions'
CatalogContent:
  - 'learn-php'
  - 'paths/computer-science'
---

> The **`deg2rad()`** function converts the number in degrees to the radian equivalent.

Reviewer 2: ```suggestion
The **`deg2rad()`** function converts a number in degrees to the radian equivalent.
```

> The result returns a value type of `float`.

Reviewer: Can we move this line to the end of the syntax section? (Inserted before line 28)

Author: Done

```pseudo
$result = deg2rad(n);
```

Where `n` is the number in degrees to be converted to the radian equivalent.

The result returns a value type of `float`.

## Example

```php
echo deg2rad(90);

> // Output: 1.5707963267949

Reviewer 2: ```suggestion
```

```

> The result will be a float value of `1.5707963267949` as the function converts degree `90` to the radian equivalent.

Reviewer 2: ```suggestion
The code above will result in the following output:
```

> 

Reviewer 2: ```shell
1.5707963267949
```

> The result will be a float value of `1.5707963267949` as the function converts degree `90` to the radian equivalent `1.5707963267949`.

Author: Done

> ## Codebyte Example

Reviewer 2: I would recommend deleting the codebyte because it's not really adding any information to this entry. Ideally every example is illustrating some specific functionality that isn't demonstrated in the others. Generally, we shouldn't have examples that are duplicative. Sometimes in entries that cover many elements, such as an entry on conditionals, we may have a summary section that has examples similar to those presented earlier.
